tests/test_namespaces.py

```diff
@@ -72,6 +72,7 @@ def create_namespace(stub, rbd_pool, rbd_image, nsid):
                                           rbd_image_name=rbd_image,
                                           nsid=nsid,
                                           block_size=4096,
+                                          load_balancing_group=1,
```

Comment:
@leonidc this is wrong, you should use "anagrpid" as the field name and not "load_balancing_group". This is how the namespace_add_req structure us defined.